### PR TITLE
fix: add numpy to pipfile

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**

### CODE REVIEW

**Review by jrobinAV — COMMENTED** [review]
Looks good to me.
 Is it still a draftt?

File: Pipfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ marshmallow = "==3.20.1"
 networkx = "==2.6"
 openpyxl = "==3.1.2"
 pandas = "==1.3.5"
+numpy = "<2.0.0"
```
Comment (trgiangdo) [inline Pipfile]: Should we put this under "dev-packages"?
Just to indicate that this condition is only used when developing taipy, not an actual taipy requirement.

**Review by trgiangdo — DISMISSED** [review]
Minor suggestion. Other than that, the change looks good to me

File: Pipfile (hunk 1 above)
Comment (joaoandre-avaiga) [reply Pipfile]: I like it. Changing it now

**Review by trgiangdo — APPROVED** [review]

**Review by toan-quach — DISMISSED** [review]

**Review by trgiangdo — CHANGES_REQUESTED** [review]
The coverage test is failing. Probably because it's not using the same Pipfile?

**Review by jrobinAV — DISMISSED** [review]

**Comment by toan-quach** [comment]
I have the same thinking with @trgiangdo but I just checked and not sure what the problem is 🤣

**Review by jrobinAV — APPROVED** [review]

**Review by trgiangdo — APPROVED** [review]

**Pull request merged**